rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -202,6 +202,10 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 	m_emu_settings->EnhanceCheckBox(ui->accurateXFloat, emu_settings_type::AccurateXFloat);
 	SubscribeTooltip(ui->accurateXFloat, tooltips.settings.accurate_xfloat);
 
+	m_emu_settings->EnhanceCheckBox(ui->fullWidthAVX512, emu_settings_type::FullWidthAVX512);
+	SubscribeTooltip(ui->fullWidthAVX512, tooltips.settings.full_width_avx512);
+	ui->fullWidthAVX512->setDisabled(!utils::has_avx512());
```

Comment:
Can just say setEnabled without the !.
But I guess that's preference

Reply:
Theyn you'll have to set the default behavior to disabled, no?

Reply:
Oh, I didn't notice that there's a setEnabled. I was looking just looking at the LLVMdfma option. I'll just use that instead then